content/doc/book/system-administration/admin-password-reset.adoc

```diff
@@ -0,0 +1,32 @@
+---
+layout: section
+---
+ifdef::backend-html5[]
+:notitle:
+:description:
+:author:
+:email: jenkinsci-users@googlegroups.com
+:sectanchors:
+:toc: left
+endif::[]
+
+==Admin-Password-Reset Jenkins
+As a system administrator, you may need to reset the Jenkins admin password if you have lost or forgotten it. To cover this possibility we have steps to follow to reset the admin password.
+
+==Steps to reset Jenkins Admin Password
+* Login to your Jenkins server and edit the Jenkins configuration file(config.xml) inside your Jenkins folder.
+* Look for the **useSecurity** option and change it from **true** to **false**.
+* Save your file and close it.
+* Restart Jenkins service to apply changes.(This time it will not ask for password)
+* On the dashboard click on **Manage Jenkins** option on the left side.
+* In Manage Jenkins , in the **Security** section you will get **Configure Global Security**. Click on it.
+* In **Security Realm**, click on the checkbox of **Jenkins' own user database** and save it.You will get redirected to the Manage Jenkins page.
+* In **Security**, click on **Manage Users**. 
+
+You will get a screen showing your user name with multiple tabs . Click on the setting icon.
+* Search for **Password** section to change your password.
+After changing the password you will can try to login again for you Jenkins and give the username and password that you have reseted.
+
+== Video link:  
+link:https://www.youtube.com/watch?v=_VhOMyWDIcY
+===Thank You 
```

Comment:
```suggestion
link:https://www.youtube.com/watch?v=_VhOMyWDIcY
```
Thank you is unnecessary in this case, so removing it and the section. Also, the video link should not be separated into its own section like that. It can be included in the body of the documentation towards the end, or towards the beginning where it may be more useful. The video can help users _before_ they read through all of the documentation, so having it towards the top of the page/above the page fold, would be beneficial.